### DISCUSSION TITLE
All connection lost after update from 6.x.x to 7.0.5

### DISCUSSION THREAD

**Discussion by supiik** [dbeaver/dbeaver#8747] [Q&A] [answered]
I have updated from 6.x.x to 7.0.5
DBeaver does not start and I get attached error:

[1589915420865.log](https://github.com/dbeaver/dbeaver/files/4652696/1589915420865.log)

If I start it with sudo it starts, but I lost all my connection setting. This is bad, this is bad.

**Comment by serge-rider** [comment, marked as answer]
Perhaps you did run dbeaver under root permissions and it created a set of working files right in /usr/shared folder.

How to fix: remove old dbeaver installation under sudo.
Reinstall dbeaver and run it under regular user.
